packages/net-stubbing/lib/server/util.ts

```diff
@@ -241,7 +241,8 @@ export function getBodyEncoding (req: CyHttpMessages.IncomingRequest): BodyEncod
 
   // a simple heuristic for detecting UTF8 encoded requests
   if (req.headers && req.headers['content-type']) {
-    const contentType = req.headers['content-type'].toLowerCase()
+    const contentTypeHeader = req.headers['content-type'] as string
+    const contentType = contentTypeHeader.toLowerCase()
```

Comment:
```suggestion
    const contentTypeHeader = req.headers['content-type'] as string
    const contentType = contentTypeHeader.toLowerCase()
```

ehh, this is not quite right, the request will error out if for example it passes two `content-type` headers:

```
content-type: foo
content-type: bar
```

...`.toLowerCase` will not exist on the `Array` type.

but hey, it's okay, since that's not valid content-type according to the HTTP spec.